office365-x64/README.md

```diff
@@ -0,0 +1,30 @@
+# Visual Studio 2017 Professional choco package
```

Comment:
@rismoney @ferventcoder think this was meant to be Office 365, right?

Reply:
I saw that later, but yes I believe that is true.